guides/common/modules/proc_adding-a-bmc-interface.adoc

```diff
@@ -1,30 +1,45 @@
 [id="Adding_a_Baseboard_Management_Controller_Interface_{context}"]
 = Adding a baseboard management controller (BMC) interface
 
+{Project} can control the power status of bare-metal hosts.
 Use this procedure to configure a baseboard management controller (BMC) interface for a host that supports this feature.
 
 .Prerequisites
-* The `ipmitool` package is installed.
-* You know the MAC address, IP address, and other details of the BMC interface on the host, and the appropriate credentials for that interface.
+* You know the MAC address, IP address, and other details of the BMC interface on the host, and authentication credentials for that interface.
 +
 [NOTE]
 ====
 You only need the MAC address for the BMC interface if the BMC interface is managed, so that it can create a DHCP reservation.
 ====
 
 .Procedure
-. Enable BMC on the {SmartProxy} server if it is not already enabled:
-.. Configure BMC power management on {SmartProxyServer} by running the `{foreman-installer}` script with the following options:
+. Install `ipmitool` on your {SmartProxy}:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# {foreman-installer} --foreman-proxy-bmc=true \
---foreman-proxy-bmc-default-provider=ipmitool
+# {project-package-install} ipmitool
 ----
-.. In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}*.
-.. From the list in the *Actions* column, click *Refresh*.
-The list in the *Features* column should now include BMC.
-. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
+. Enable BMC power management on your {SmartProxy}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} \
+--foreman-proxy-bmc-default-provider=ipmitool \
+--foreman-proxy-bmc=true
+----
+. In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}*.
+. Optional: From the list in the *Actions* column, select *Refresh*.
```

Comment:
I'd like to avoid telling users to refresh. If anything, flip it around: Verify BMC is present, otherwise refresh.

Reply:
Dropped!